docs/src/index.md

```diff
@@ -146,6 +146,7 @@ F-test: 2 models fitted on 50 observations
 Many of the methods provided by this package have names similar to those in [R](http://www.r-project.org).
 - `coef`: extract the estimates of the coefficients in the model
 - `deviance`: measure of the model fit, weighted residual sum of squares for lm's
+- `residuals`: returns the vector of residuals. This yields the same vector as `@. sign(y - predict(model)) * sqrt(devresid(d, y, predict(model)))`, where `d` is the distribution object that was used with the original `lm` call. 
```

Comment:
I'd just use a short description for consistency with other functions in the list:
```suggestion
- `residuals`: return the vector of residuals
```
Also probably better put it after `predict`, since these are quire similar.

If you feel like adding more information, it could go to a docstring instead.

Reply:
I've taken on board your suggestion for the `residuals` stuff under the `Many of the methods` heading. I've also moved the note about how `residuals` returns the same thing as the longer code block involving `devresid` to the docstring for `devresid` (since that's probably where the question will come up for folks), though feel free to disregard that if you don't think it'll be useful.